app/config/config.yml

```diff
@@ -96,13 +96,19 @@ twig:
 api_platform:
   title: Backoffice API
   version: 0.1.0
+  enable_docs: true
   enable_entrypoint: false
+  enable_swagger: true
+  enable_swagger_ui: true
```

Comment:
Shouldn't we condition this as dev mode

Reply:
I wondered the same thing actually! But I'm not sure, since the doc can be useful when you set up a server-to-server configuration if you need to check the API even on a shop on production 🤔

I'd like the opinion of others from the team, I don't mind enabling it only on dev mode In which case I will also condition the small block to help access it only visible in dev mode as well

Reply:
Is access to the swagger interface secure?

Reply:
I think you should enable it even in production mode. It can be useful for external developers.

Reply:
Ok so let's keep it in prod mode, and @M0rgan01 yes you can't access the doc unless you're logged in to the BO, that's mostly why I let it in prod mode